lib/editor/selectors/timetable.js

```diff
@@ -154,6 +154,8 @@ const isCellValueInvalid = (
   tripIds: Array<string>
 ): ?EditorValidationIssue => {
   const col = columns[colIndex]
+  const isFlexStopTime = (col: TimetableColumn) => col.key.includes('startPickupDropoffWindow') || col.key.includes('endPickupDropoffWindow')
```

Comment:
Any specific reason why this has to be a function?
- If so, move the function to outside `isCellValueInvalid`.
- If not, change to a variable and move it to above line 177.

Reply:
Kept it as a function because it feels like it may be reused later. I moved it outside `isCellValueInvalid` in https://github.com/ibi-group/datatools-ui/pull/884/commits/aeb4c628a03ad20c4a3116f9525609f025d3f32f